custom/aaa/static/aaa/spec/models/child.spec.js

```diff
@@ -0,0 +1,183 @@
+/* global sinon, moment */
+
+var pageData = hqImport('hqwebapp/js/initial_page_data');
+
+describe('Child models', function () {
+    var childModels, reachUtils, clock;
+
+    beforeEach(function () {
+        childModels = hqImport('aaa/js/models/child');
+        reachUtils = hqImport('aaa/js/utils/reach_utils');
+        pageData.registerUrl('unified_beneficiary_details', 'unified_beneficiary_details/details_type/beneficiary_id/');
+        var today = moment('2020-01-01').toDate();
+        clock = sinon.useFakeTimers(today.getTime());
+    });
+
+    afterEach(function () {
+        clock.restore();
+    });
+
+    describe('child config model', function () {
+        it('test model properties', function () {
```

Comment:
Not necessarily actionable for these tests, but something to keep in mind in the future:

When writing test names in javascript they should be able to be read as a sentence when concatenating describe and it commands. For example in the mocha docs https://mochajs.org/#getting-started their example test can be read as "Array #indexOf() should return -1 when the value is not present" To follow that sort of example, this test could be broken into several tests:

"Child UBR list view should include name"
"Child UBR list view should include age"
"Child UBR list view should include gender"
and so on

Reply:
@emord I will update test names in meantime